README.md

```diff
@@ -46,7 +46,14 @@ Basic build steps:
 Please do *NOT* use an editor that automatically reformats.
 
 Use the coding style set by astyle (http://astyle.sourceforge.net/) with the following parameteres:
-> astyle --style=stroustrup --indent-switches --indent-labels --pad-oper --pad-header --align-pointer=name --add-brackets --convert-tabs --max-code-length=90 --break-after-logical --suffix=none *.c *.h --recursive --exclude=src/yajl --exclude=tests/windows/hidapi
+
+    astyle --style=stroustrup --indent-switches --indent-labels --pad-oper --pad-header --align-pointer=name --add-brackets --convert-tabs --max-code-length=90 --break-after-logical --suffix=none *.c *.h --recursive --exclude=src/yajl --exclude=tests/windows/hidapi
```

Comment:
I changed the Markdown from comment (> ) to code (/tab), because we have it like that in the other commands listed in the readme. See formatted result: https://github.com/lclc/mcu/blob/master/README.md